### PR TITLE
Update product names

### PR DESCRIPTION
**Pull request by FSM1**
Update product names to align with the names on invoices, as well as add production product id's to the lookup

### CODE REVIEW

File: packages/files-ui/src/Contexts/BillingContext.tsx [hunk 1]
```diff
@@ -39,17 +39,31 @@ const ProductMapping: {[key: string]: {
...
   prod_JwSGHB8qFx7rRM: {
-    name: t`Premium plan`,
-    description: t`Premium plan`
+    name: t`Files Max`,
+    description: ""
+  },
+  // Production Product Ids
+  prod_KRAq3CngQMKebw: {
+    name: t`Files Free`,
+    description: ""
```
Comment (Tbaut) [inline packages/files-ui/src/Contexts/BillingContext.tsx]: Side comment, I wonder what the description is supposed to be here, it's empty for all..

**Review by Tbaut — APPROVED** [review]

File: packages/files-ui/src/Contexts/BillingContext.tsx (hunk 1 above)
Comment (FSM1) [reply packages/files-ui/src/Contexts/BillingContext.tsx]: also do not think it is used anywhere. Think we can nuke it...

**Comment by Tbaut** [comment]
Tests are failing, looks like it's pretty consistent

**Comment by FSM1** [comment]
> Tests are failing, looks like it's pretty consistent

yeah I am fixing these slowly... some of the selectors were using the plan names.

**Comment by asnaith** [comment]
> > Tests are failing, looks like it's pretty consistent
> 
> yeah I am fixing these slowly... some of the selectors were using the plan names.

@FSM1 Ah yes, I know that part you are referring to. Unfortunately using some of the plan name text was a necessity in order to differentiate between all the plan containers and ensure we select a specific select plan button. 

If there's a way of assigning a unique identifier to each plan container that would be better but I wasn't able to find a way to do that.

**Comment by tanmoyAtb** [comment]
Hey @FSM1 some of the `description` fields are still around. Not sure if it's still in progress.

**Comment by FSM1** [comment]
> Hey @FSM1 some of the `description` fields are still around. Not sure if it's still in progress.

These have been removed from the mapping, and the processing of the various products, but `product.description` is still returned by the API so might appear elsewhere in the code.

**Review by tanmoyAtb — APPROVED** [review]
Looks good! 
On that note, we can also make the changes on storage side if needed. Storage has the old names and description field.

**Comment by FSM1** [comment]
> Looks good! On that note, we can also make the changes on storage side if needed. Storage has the old names and description field.

Will leave that for later, when the storage product names are finalized and ready...

**Review by asnaith — APPROVED** [review]
Thanks for all the test updates 🙏

**Pull request merged**